src/store/getters.ts

```diff
@@ -201,11 +201,11 @@ export default {
       return account;
     };
   },
-  assetFees(...context: GetterContext) {
+  getSuggestedFeePrices(...context: GetterContext) {
```

Comment:
The getters are not usually named this way?
We should be consistent

Reply:
Removing leading `get` out of the method's name.